gpu-operator/precompiled-drivers.rst

```diff
@@ -59,9 +59,9 @@ Limitations and Restrictions
   refer to :external+ocp:doc:`gpu-operator-with-precompiled-drivers`.
 
 * NVIDIA supports precompiled driver containers for the most recently released long-term
-  servicing branch (LTSB) driver branch, 525.
+  servicing branch (LTSB) driver branch.
 
-* NVIDIA builds images for the ``generic`` kernel variant.
+* NVIDIA builds images for the ``aws``, ``azure``, ``generic``, ``nvidia``, and ``oracle`` kernel variants.
```

Comment:
I would prefer using the term flavour/flavor here.  As per this [link](https://ubuntu.com/kernel/variants), the suffix is the flavour, but the whole kernel version string is called the variant
```suggestion
* NVIDIA builds images for the ``aws``, ``azure``, ``generic``, ``nvidia``, and ``oracle`` kernel variants.
```

Reply:
"Flavour" appears once on that page and really seems like a colloquialism.  I can't count how many times "variant" is used and the table of Current variant kernels shows the aws and azure strings.  I'd really prefer to continue with variant.